themes/bootstrap3/js/account_ajax.js

```diff
@@ -223,86 +223,90 @@ $(document).ready(function registerAccountAjax() {
     }
   });
 
-  function statusIcon(icon, className, title) {
-    return VuFind.icon(icon, {
-      class: className,
-      "data-toggle": "tooltip",
-      "title": VuFind.translate(title),
-    });
-  }
-
   VuFind.account.register("holds", {
     selector: ".holds-status",
     ajaxMethod: "getUserHolds",
     render: function render($element, status, ICON_LEVELS) {
+      var html = '';
       var level = ICON_LEVELS.NONE;
       if (status.available > 0) {
-        $element.html(
-          statusIcon("my-account-notification", "text-success", "account_requests_available")
-        );
+        html += '<span class="badge ok" data-toggle="tooltip" title="' + VuFind.translate('account_requests_available') + '">' + status.available + '</span>';
         level = ICON_LEVELS.GOOD;
-      } else if (status.in_transit > 0) {
-        $element.html(
-          statusIcon("my-account-warning", "text-warning", "account_requests_in_transit")
-        );
+      }
+      if (status.in_transit > 0) {
+        html += '<span class="badge warn" data-toggle="tooltip" title="' + VuFind.translate('account_requests_in_transit') + '">' + status.in_transit + '</span>';
+      }
+      if (status.other > 0) {
+        html += '<span class="badge overdue" data-toggle="tooltip" title="' + VuFind.translate('account_requests_other') + '">' + status.other + '</span>';
```

Comment:
I wonder if we should create a style with a more generic name than `overdue`, since `overdue` isn't really accurate in this situation. Maybe something like `alert`?

Reply:
@aidaluuppala Good idea, this! We use something more of a flat design and would much prefer the item numbers over the bell icon. The numbers convey much more information than the bell and the entire menu would look more consistent.
![my-account-bell](https://user-images.githubusercontent.com/8236627/177115049-72bda77d-ad44-465a-901c-25a6f88f9813.png)
.